trinity/plugins/registry.py

```diff
@@ -35,10 +35,14 @@ def is_ipython_available() -> bool:
         return True
 
 
-BUILTIN_PLUGINS = (
+COMMON_BUILTIN_PLUGINS: Tuple[BasePlugin, ...] = (
     AttachPlugin() if is_ipython_available() else AttachPlugin(use_ipython=False),
```

Comment:
I just realized this can be simplified to `AttachPlugin(use_ipython=is_ipython_available())`

Reply:
Ah, yes. I think at one point that were two different classes and then later got refactored to a constructor parameter again but left the `if/else` in place. Will change it.